frontend/src/components/ListDetail.js

```diff
@@ -1,25 +1,28 @@
 import { useEffect, useState } from 'react';
-import { addItem } from "../http/lists";
+import { addItem, setCompletion } from "../http/lists";
+import { Item } from './Item';
 
-export const ListDetail = ({ selectedList }) => {
+export const ListDetail = ({ id, name, completed, items }) => {
     const [messageState, setMessageState] = useState('');
     const [errorMessageState, setErrorMessageState] = useState('');
-    const [items, setItems] = useState([])
+    const [itemsState, setItemsState] = useState(items ? items : []);
     const [newItemName, setNewItemName] = useState('');
+
     // takes an input value and adds it to the selectedList when enter is pressed
     const handleAddItem = async (event) => {
         if (event.charCode === 13) {
             if (!newItemName) {
                 setErrorMessageState('Item name cannot be empty');
                 return;
             }
-            const res = await addItem(selectedList.id, newItemName);
+            const res = await addItem(id, newItemName);
             if (res.error) {
                 setErrorMessageState(res.error);
             } else {
                 setErrorMessageState('');
                 setMessageState('Task Added Successfully');
-                setItems([...items, res])
+                setItemsState([...itemsState, res])
+                console.log(items)
```

Comment:
Just a comment ( no need to make any change ): In a future PR we should go through and clean up the console log statements